src/bitcoin-cli.cpp

```diff
@@ -411,18 +413,41 @@ static int CommandLineRPC(int argc, char *argv[])
         }
         std::string rpcPass;
         if (gArgs.GetBoolArg("-stdinrpcpass", false)) {
+            NO_STDIN_ECHO();
+            if (!StdinReady()) {
+                fprintf(stderr, "RPC password> ");
```

Comment:
you could  use fputs here (same below) and avoid introducing the locale dependency exception

Reply:
`tfm::format(std::cerr, ...` should work as well

Reply:
D'oh, yeah. Removing linter changes and using `fputs`.